nodejs/packages/layer/src/wrapper.ts

```diff
@@ -201,6 +201,7 @@ const lambdaAutoInstrumentConfig: AwsLambdaInstrumentationConfig = {
     }
     return otelContext.active();
   },
+  payloadSizeLimit: OTEL_PAYLOAD_SIZE_LIMIT,
```

Comment:
Some context about this one liner helps would be nice, but approving anyway.

Reply:
This is a continuation of https://github.com/coralogix/opentelemetry-js-contrib/pull/8

OTEL_PAYLOAD_SIZE_LIMIT is already used to limit some attributes that are produced by code located in `opentelemetry-lambda`. Now I also propagate it to the code of `opentelemetry-js-contrib` where the `AwsLambdaInstrumetation` is located. And there I use it to limit the size of an attribute that contains the response body (or now, the first 50kiB of the response body)